src/ethereum/shanghai/fork.py

```diff
@@ -613,6 +614,14 @@ def process_transaction(
     output = process_message_call(message, env)
 
     gas_used = tx.gas - output.gas_left
+        
+    floor = tokens_in_calldata * TOTAL_COST_FLOOR_PER_TOKEN + TX_BASE_COST
```

Comment:
`STANDARD_TOKEN_COST` and `TOTAL_COST_FLOOR_PER_TOKEN` need to be imported from `.fork_types`.

Reply:
Thanks!
I replaced it for the imports 
```
 TX_DATA_COST_PER_NON_ZERO,
 TX_DATA_COST_PER_ZERO,
```